docs/mkdocs.yml

```diff
@@ -29,20 +29,55 @@ theme:
     - navigation.instant
     - navigation.tabs
     - navigation.sections
-    - toc.integrate
     - navigation.top
 
 nav:
-- webknossos Python API:
-  - Intro: webknossos/index.md
+- webKnossos:
```

Comment:
The index? Not out of the box, but this would work with something like
* https://github.com/backstage/mkdocs-monorepo-plugin
* https://github.com/lukasgeiter/mkdocs-awesome-pages-plugin